tock/tock/templates/projects/project_detail.html

```diff
@@ -96,19 +96,4 @@ <h3 class="chart__title">Latest Periods with Hours Logged</h3>
   </tbody>
 </table>
 
-<link rel="stylesheet" href="{% static 'css/vendor/jquery.dataTables.min.css' %}" />
-<script src="{% static 'js/vendor/jquery.dataTables.min.js' %}"></script>
-<script src="{% static 'js/vendor/dataTables.fixedColumns.min.js' %}"></script>
-<script type="text/javascript">
-$(document).ready(function() {
-  var $table = $('#dataTable').dataTable({
-    scrollX: true,
-    paging: false,
-    lengthChange: false
```

Comment:
This line contained a syntax error which has prevented datatables from running. It doesn't look like this has worked for quite some time as resolving the syntax breaks styling on the page. Herein we remove in favor of fixing.

Reply:
maybe this is an opportunity to remove the datatable dependency altogether?

I'd love it if we could get to a point where we didn't have a jquery dependency, either. This might be one step in that direction?

Reply:
We are 😄, that code snippet and the source files associated with datatables are removed w/ this PR.